modules/logging-release-notes-5-6-15.adoc

```diff
@@ -0,0 +1,25 @@
+//module included in logging-5-6-release-notes.adoc
+:content-type: REFERENCE
+[id="logging-release-notes-5-6-15"]
+= Logging 5.6.15
+This release includes link:https://access.redhat.com/errata/RHBA-2024:0270[OpenShift Logging Bug Fix Release 5.6.15].
+
+[id="logging-release-notes-5-6-15-known-issue"]
+== Known issue
+Currently, the log alert is not available on an IPv6-enabled cluster. There is currently no workaround for this issue. (link:https://issues.redhat.com/browse/LOG-4892[LOG-4892])
```

Comment:
As mentioned elsewhere, the new text is available on https://issues.redhat.com/browse/LOG-4892